docs/src/pages/FlatListPage/FlatListPage.js

```diff
@@ -16,6 +16,26 @@
  * limitations under the License.
  */
 
-import page from './AndroidFlarePage';
```

Comment:
Is this page renaming and change expected seems a bit strange going from a Flare page to FlatListPage?

Reply:
I think this is just git being dumb and thinking that I renamed this file when actually I deleted it and added another

Reply:
👍